src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -62,39 +62,39 @@ export type ModelingCommandSchema = {
   Extrude: {
     // Enables editing workflow
     nodeToEdit?: PathToNode
-    selection: Selections // & { type: 'face' } would be cool to lock that down
-    // result: (typeof EXTRUSION_RESULTS)[number]
-    distance: KclCommandValue
+    // KCL stdlib arguments
+    sketches: Selections
+    length: KclCommandValue
   }
   Sweep: {
     // Enables editing workflow
     nodeToEdit?: PathToNode
-    // Arguments
-    target: Selections
-    trajectory: Selections
-    sectional: boolean
+    // KCL stdlib arguments
+    sketches: Selections
+    path: Selections
+    sectional?: boolean
```

Comment:
Quick note: taking the chance to make sectional optional here, instead of writing the default value to kcl everytime

Reply:
Nice we should make it one of the first truly optional arguments like these ones with the `+`s here
![Screenshot 2025-05-06 at 3 03 16 PM](https://github.com/user-attachments/assets/6ce70c95-4bf9-4f70-a9a1-2e787a92da98)

Reply:
(later)

Reply:
Oh yesssss